barretenberg/cpp/src/barretenberg/eccvm/eccvm_flavor.hpp

```diff
@@ -122,91 +128,91 @@ class ECCVMFlavor {
     template <typename DataType> class WireEntities {
       public:
         DEFINE_FLAVOR_MEMBERS(DataType,
-                              transcript_add,                              // column 0
```

Comment:
had to sort the entities differently to get a contiguous range of repeating commitments